applications/osb-portal/src/components/workspace/drawer/WorkspaceResourceBrowser.tsx

```diff
@@ -18,11 +18,16 @@ import {
 } from "../../icons";
 
 
+const openFileResource = (resource: WorkspaceResource) => (e: any) => {
+  const fileName = resource.location.slice(resource.location.lastIndexOf("/")+1);
+  const iFrame: HTMLIFrameElement = document.getElementById("workspace-frame") as HTMLIFrameElement;
+  iFrame.contentWindow.postMessage(fileName, '*');
+}
 
 const OSBTreeItem = (props: { resource: WorkspaceResource }) => {
   const { resource } = props;
   return (
-    <Box display="flex" alignItems="center" justifyContent="space-between">
+    <Box display="flex" alignItems="center" justifyContent="space-between" onClick={openFileResource(resource)} >
```

Comment:
Need to disable the click for resources in case that:
* resource is not ready 
* resource is mapped to a different application (need to think more about what we really want to do in that case)

Reply:
ah yeah okay will change that

Reply:
for now the implementation of the "different" application is just the same as for NWB --> so onclick == send postmessage
when we need other functionality for other apps then please create a new issue

Reply:
I agree with using the same kind of messaging for other applications. I'm also thinking about the case where we opened nwb explorer and then click on a netpyne resource. I'd like to avoid at least errors for now by simply disabling it, then we can tdiscuss about a proper strategy after everyone starts using the basic version.